src/test/java/com/dsrm/dsrmbackend/UserControllerIntegrationTests.java

```diff
@@ -87,7 +89,8 @@ void addValidUser() throws Exception {
         assertEquals("Jan", user.getName());
         assertEquals("Jan@gmail.com", user.getEmail());
         assertEquals("Chraboszcz", user.getSurname());
-        assertEquals("Marciniak", user.getPassword());
+        PasswordEncoder encoder = new BCryptPasswordEncoder();
+        assertTrue(encoder.matches(userRequestDTO.getPassword(), user.getPassword()));
```

Comment:
I would prefer to match this against string from database. I would not generate password in tests

Reply:
I think it matches password against password from database. But about password generation in test - am I supposed to remove those lines of password matching?

Reply:
No I meant that you should match against hardcoded strings like ```$2a$10$MXlOV6auIzKB6GTzcpRZsu7yoBybFpX8OP4pQY.Y.NhjUBGIqZl76``` to check if password is encoded correctly